### PR TITLE
feat: set private labeling setting on project

### PR DESCRIPTION
**Pull request by fgerschwiler**

### CODE REVIEW

File: internal/auth/repository/eventsourcing/eventstore/auth_request.go [hunk 1]
```diff
@@ -482,9 +485,16 @@ func (repo *AuthRequestRepo) fillPolicies(ctx context.Context, request *domain.A
 	if orgID == "" {
 		orgID = request.UserOrgID
 	}
+	privateLabelingOrgID := orgID
+	if privateLabelingOrgID == "" {
+		privateLabelingOrgID = request.RequestedPrivateLabelingOrgID
+	}
```
Comment (livio-a) [inline internal/auth/repository/eventsourcing/eventstore/auth_request.go:491]: is this order really intentioned for private labeling?
1. primary domain scope
2. org from user
3. private label scope
4. iam

File: internal/auth_request/model/request.go [hunk 2]
```diff
@@ -19,10 +19,11 @@ const (
...
 const (
-	OrgDomainPrimaryScope = "urn:zitadel:iam:org:domain:primary:"
-	OrgDomainPrimaryClaim = "urn:zitadel:iam:org:domain:primary"
-	ProjectIDScope        = "urn:zitadel:iam:org:project:id:"
-	AudSuffix             = ":aud"
+	OrgDomainPrimaryScope   = "urn:zitadel:iam:org:domain:primary:"
+	OrgDomainPrimaryClaim   = "urn:zitadel:iam:org:domain:primary"
+	OrgPrivateLabelingOrgID = "urn:zitadel:iam:org:private:labeling:id:"
+	ProjectIDScope          = "urn:zitadel:iam:org:project:id:"
+	AudSuffix               = ":aud"
```
Comment (livio-a) [inline internal/auth_request/model/request.go:26]: are they still needed? there are exactly the same values in the domain package

File: internal/domain/request.go [hunk 3]
```diff
@@ -1,10 +1,11 @@
...
 const (
-	OrgDomainPrimaryScope = "urn:zitadel:iam:org:domain:primary:"
-	OrgDomainPrimaryClaim = "urn:zitadel:iam:org:domain:primary"
-	ProjectIDScope        = "urn:zitadel:iam:org:project:id:"
-	AudSuffix             = ":aud"
+	OrgDomainPrimaryScope   = "urn:zitadel:iam:org:domain:primary:"
+	OrgDomainPrimaryClaim   = "urn:zitadel:iam:org:domain:primary"
+	OrgPrivateLabelingScope = "urn:zitadel:iam:org:private:labeling:id:"
+	ProjectIDScope          = "urn:zitadel:iam:org:project:id:"
+	AudSuffix               = ":aud"
```
Comment (livio-a) [inline internal/domain/request.go:8]: we should quickly discuss how we want to define out scopes

the schemas of the org scopes differ from the project scope... id at the end or in between with usage at the end

File: internal/ui/login/handler/renderer.go [hunk 4]
```diff
@@ -422,6 +423,19 @@ func (l *Login) getOrgID(authReq *domain.AuthRequest) string {
...
+	if authReq == nil {
+		return ""
+	}
+	if authReq.RequestedOrgID != "" {
+
+	}
+	if authReq.UserOrgID != "" {
+		return authReq.UserOrgID
+	}
+	return authReq.RequestedPrivateLabelingOrgID
```
Comment (livio-a) [inline internal/ui/login/handler/renderer.go:436]: still unsure about the order and return is missing

File: internal/auth/repository/eventsourcing/eventstore/auth_request.go (hunk 1 above)
Comment (fgerschwiler) [reply internal/auth/repository/eventsourcing/eventstore/auth_request.go:491]: yes, do you think it should be in another order?

File: internal/auth_request/model/request.go (hunk 2 above)
Comment (fgerschwiler) [reply internal/auth_request/model/request.go:26]: I did it like with the primary domain scope. i don't know why we have it in two packages. but at the moment both are in use

File: proto/zitadel/management.proto [hunk 5]
```diff
@@ -990,7 +990,8 @@ service ManagementService {
     }
 
     // Adds an new project to the organisation
-    rpc AddProject(AddProjectRequest) returns (AddProjectResponse) {
+    rpc AddProject(AddProjectRequest
+        ) returns (AddProjectResponse) {
```
Comment (livio-a) [inline proto/zitadel/management.proto:994]:
```suggestion
    rpc AddProject(AddProjectRequest) returns (AddProjectResponse) {
```

File: migrations/cockroach/V1.63__project_private_labeling.sql [hunk 6]
```diff
@@ -0,0 +1,5 @@
+ALTER TABLE management.projects ADD COLUMN private_labeling_setting SMALLINT;
```
Comment (livio-a) [inline migrations/cockroach/V1.63__project_private_labeling.sql:1]: rename to V1.67__project_private_labeling.sql

**Review by livio-a — CHANGES_REQUESTED** [review]
only small changes... but can you quickly discuss the differences between the settings

**Review by livio-a — APPROVED** [review]

**Pull request merged**